Release workflow fails to collect .deb and .exe artifacts

The `create-release` job in `.github/workflows/on-tag.yml` fails at the "Collect and rename artifacts" step:

```
unzip:  cannot find or open staging/deb/*.zip, staging/deb/*.zip.zip or staging/deb/*.zip.ZIP.
No zipfiles found.
##[error]Process completed with exit code 9.
```

`actions/download-artifact@v4` extracts the artifact archive, so `staging/deb/` contains the `.deb` and `staging/win/` the `.exe` directly — there is nothing to unzip. This has broken the draft release for 0.40.0, 0.40.1 and 0.41.0.

--- a/.github/workflows/on-tag.yml
+++ b/.github/workflows/on-tag.yml
@@ -161,8 +161,8 @@ jobs:
           TAG="${{ env.RELEASE_TAG }}"
           mkdir -p release-artifacts
           mv staging/shadow/*.zip "release-artifacts/sysapi-shadow-${TAG}.zip"
-          unzip -j staging/deb/*.zip -d release-artifacts/
-          unzip -j staging/win/*.zip -d release-artifacts/
+          mv staging/deb/*.deb release-artifacts/
+          mv staging/win/*.exe release-artifacts/
       - name: Create draft release
         run: |
           TAG="${{ env.RELEASE_TAG }}"

